src/Utilities/Utils.res

```diff
@@ -57,6 +57,26 @@ let getInt = (dict, key, default: int) => {
   ->Belt.Float.toInt
 }
 
+let getFloatFromString = (str, default) => {
+  let val = str->Js.Float.fromString
+  val->Js.Float.isNaN ? default : val
+}
+
+let getFloatFromJson = (json, default) => {
```

Comment:
There is already a function called `getFloatfromjson` in OrcaUtils. Could you please unify the two

Reply:
Updated. Removed from OrcaUtils